src/xdp-context.c

```diff
@@ -44,21 +44,21 @@
 #include "network-monitor.h"
 #include "notification.h"
 #include "open-uri.h"
-#include "xdp-permissions.h"
 #include "power-profile-monitor.h"
 #include "print.h"
 #include "proxy-resolver.h"
 #include "realtime.h"
 #include "registry.h"
 #include "remote-desktop.h"
-#include "xdp-request.h"
 #include "screen-cast.h"
 #include "screenshot.h"
 #include "secret.h"
 #include "settings.h"
 #include "trash.h"
 #include "usb.h"
 #include "wallpaper.h"
+#include "xdp-permissions.h"
+#include "xdp-request.h"
```

Comment:
Since you're reordering these, why can't they go into the list of other xdp headers?